proto-src/psdbconnect/v1alpha1/connect.proto

```diff
@@ -2,6 +2,7 @@ syntax = "proto3";
 
 package psdbconnect.v1alpha1;
 
+import "vitess/binlogdata/v18/binlogdata.proto";
```

Comment:
We would need this to match also v19.

Reply:
I missed that it was available on the v19 protos too. Updated!